docs/unified/protectli/building-manual.md

```diff
@@ -0,0 +1,373 @@
+# Building manual
+
+=== "fw6"
+
+    ## Requirements
+
+    - Docker
+        + follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+        + follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+    - Git
+        + `sudo apt-get install git`
+
+    ## Building
+
+    To build Dasharo firmware image, follow the steps below:
+
+    1. Clone the coreboot repository:
+
+        ```bash
+        git clone https://github.com/Dasharo/coreboot.git -b protectli_vault_kbl/release
+        ```
+
+    To build a specific version replace `protectli_vault_kbl/release` to
+    `protectli_vault_kbl_v1.0.x` where `x` is the version number.
+
+    2. Start build process (note it requires certain blobs to proceed):
+
+        ```bash
+        cd coreboot
+        git submodule update --init --checkout
+        # you will need to obtain the ZIP with blobs at this point
+        unzip protectli_blobs.zip -d 3rdparty/blobs/mainboard
+        ./build.sh fw6
+        ```
+
+=== "v1x1x"
+
+    ## Intro
+
+    This document describes the procedure for compiling coreboot for Protectli
```

Comment:
This can go on top-level section of the document, simply saying it explains building Dasharo for all Protectli boards, right?

We need a bit more work towards marking this document more generic, not simply moving individual files into tabs.